ui/app/AppLayouts/Onboarding/views/InsertDetailsView.qml

```diff
@@ -54,8 +54,10 @@ Item {
     }
 
     ColumnLayout {
-        anchors.centerIn: parent
-        spacing: Style.current.padding
+        height: 461
```

Comment:
This looks like a funny value, I'd just go with 460 then, but I'm sure you have your reasons :D